generic/tests/cfg/netperf.cfg

```diff
@@ -77,6 +77,7 @@
     #    log_guestinfo_exec = cmd /c
     #    log_guestinfo_path = C:\log_guestinfo.bat
         server_mtu_cmd = "netsh interface ipv4 set subinterface "%s" mtu=%s"
+    windows_disable_firewall = "netsh advfirewall set allprofiles state off"
```

Comment:
I would suggest list this param under `Windows:` variants, so increase the indent of this line, and rename it to `disable_firewall_command`

Reply:
Updated, please help review.